src/features/hotspots/details/HotspotDetails.tsx

```diff
@@ -289,11 +290,13 @@ const HotspotDetails = () => {
         </BottomSheet>
       </Box>
 
-      <HotspotSettings
-        hotspot={hotspot}
-        visible={showSettings}
-        onClose={() => setShowSettings(false)}
-      />
+      <HotspotSettingsProvider>
```

Comment:
we'll need to figure out the best way to integrate these settings provider changes with the bottom sheet branch.